src/summary/level-two-routes.component.tsx

```diff
@@ -11,17 +12,23 @@ export function LevelTwoRoutes(props: LevelTwoRoutesProps) {
         <Route
           exact
           path={`/patient/:patientUuid/chart/allergy`}
-          render={routeProps => <AllergyCardLevelTwo match={props.match} />}
```

Comment:
I think there is a reason why @joeldenning started the pattern of passing the `props.match` prop, maybe we can learn more about this before removing them?

Reply:
@joeldenning , could you clarify? I thought you had told me to stop doing that pattern.

Reply:
I talked to Fatma just now about this - she understands now.

Reply:
this is okay, with the old pattern, we were passing on the old `props.match`.